test/config/utility.cc

```diff
@@ -376,6 +376,21 @@ void ConfigHelper::applyConfigModifiers() {
   config_modifiers_.clear();
 }
 
+void ConfigHelper::addRuntimeOverride(absl::string_view key, absl::string_view value) {
```

Comment:
What if we want to invoke this > 1? I think we could make this entire method just `bootstrap_.mutable_layered_runtime()->mutable_layers(0)->static_layer()->... some stuff around the mutable key/value ---`.

Reply:
Sure.  Non-obvious due to how merging went but the cors test now needs 2 overrides so now works as you requested and is verified by test.